.github/workflows/test-and-coverage.yml

```diff
@@ -20,8 +20,8 @@ jobs:
 
     - name: Test
       run: |
-        git config user.email talisman-github-actions@github.com
-        git config user.name "Talisman GH Actions User"
+        git config user.email "talisman-maintainers@thoughtworks.com"
```

Comment:
why are we configuring git with email and name ?

Reply:
without this, the git commands executed by tests fail.
and we see  `panic: commaned exited with code 128`

This is needed as long as we we create repos in test

Reply:
Okay cool